go.mod

```diff
@@ -1,12 +1,13 @@
 module github.com/aep-dev/api-linter
 
-go 1.21
+go 1.22.0
```

Comment:
This is happening because some indirect dependency requires a minimum Golang version of 1.22. This is happening across the ecosystem - the only two officially supported versions of Golang now are 1.22 and 1.23. This isn't an issue - every version of Go >= 1.21 will support this since they will automatically download the required Golang version, and this is a CLI tool as it is.